product_search/main.py

```diff
@@ -0,0 +1,53 @@
+"""Return the inventory details for an item in the product inventory"""
+
+import sys
+from product_search import (
+    validate_item,
+    inventory_details_for_user_item,
+    _style_dictionary_items_for_output,
+    return_product_details,
+)
+from data_management import access_json_data
+
+from user_input import get_item_name_from_user
+
+
+def main(filename):
+    """
+    Calls to functions to get valid user input, 
+    retrieve inventory details based on input, 
+    and return details to user
+    """
+    # input
+    data = access_json_data(sys.argv[1])
+    item = get_valid_item_input(data)
+
+    # core
+    inventory = inventory_details_for_user_item(item, data)
+    inventory_for_output = _style_dictionary_items_for_output(inventory)
```

Comment:
So `_style_dictionary_items_for_output` isn't private to _this_ module since an outside module is calling it, hence no need to make it private. A private function is one that's only called by another function _within_ the same module, ie, if `inventory_details_for_user_item` called it.